game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/BadWordsTabModel.java

```diff
@@ -4,9 +4,8 @@
 
 import javax.annotation.Nonnull;
 
-import org.triplea.http.client.moderator.toolbox.AddBadWordArgs;
 import org.triplea.http.client.moderator.toolbox.ModeratorToolboxClient;
-import org.triplea.http.client.moderator.toolbox.RemoveBadWordArgs;
+import org.triplea.http.client.moderator.toolbox.UpdateBadWordsArg;
```

Comment:
RemoveBadWordArgs and AddBadWordArgs were essentially the same. Instead of needing to ever update both it was renamed to 'UpdateBadWordsArgs' and combined.